azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/configuration/CallCompositeConfiguration.kt

```diff
@@ -41,4 +43,11 @@ internal class CallCompositeConfiguration {
     var capabilitiesChangedNotificationMode: CallCompositeCapabilitiesChangedNotificationMode? = null
     var setupScreenOptions: CallCompositeSetupScreenOptions? = null
     var identifier: CommunicationIdentifier? = null
+
+    fun setTimerConfig(timerConfig: CallDurationManager) {
```

Comment:
I do not think Configuration should have any business logic in it. It only should have a configuration objects.
Also `setTimerConfig` does not reflect what it does.